code/__DEFINES/dcs/signals/signals_mob/signals_mob_silicon.dm

```diff
@@ -2,3 +2,10 @@
 #define COMSIG_PROCESS_BORGCHARGER_OCCUPANT "living_charge"
 ///sent from borg mobs to itself, for tools to catch an upcoming destroy() due to safe decon (rather than detonation)
 #define COMSIG_BORG_SAFE_DECONSTRUCT "borg_safe_decon"
+///called from /obj/item/borg/cyborghug/attack proc for cyborg lovers trait.
+#define COMSIG_BORG_TOUCH_MOB "borg_touch_mob"
+///called from /obj/item/borg/cyborghug/attack proc for cyborg lovers trait.
+#define COMSIG_BORG_HUG_MOB "borg_hug_mob"
+	#define COMSIG_BORG_HUG_SUCCESS 1
+///called from /mob/living/silicon/attack_hand proc for cyborg lovers trait.
+#define COMSIG_MOB_PAT_BORG "mob_pat_borg"
```

Comment:
Remove "for cyborg lovers trait"--this is strictly an implementation detail.

Reply:
done